xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -576,6 +576,7 @@ void CVideoPlayerAudio::Process()
         msg.player = VideoPlayer_AUDIO;
         msg.cachetotal = cachetotal;
         msg.cachetime = cachetime;
+        msg.next_pts = m_messageQueue.GetTimeBack();
         msg.timestamp = m_audioClock;
```

Comment:
looks like you are digging in the wrong place. why do you try to"fix" a mechanism that should not be engaged for your case. you don't do seeks in the stream nor do you switch to a completely different stream. why do you want to resync a/v?